### PR TITLE
fix memory leak in produer benchmark

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
Please review @kciesielski

File: benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaProducerBenchmarks.scala [hunk 1]
```diff
@@ -29,8 +29,7 @@ object ReactiveKafkaProducerBenchmarks extends LazyLogging {
   def plainFlow(fixture: ReactiveKafkaProducerTestFixture[Int], meter: Meter)(implicit mat: Materializer): Unit = {
     logger.debug("Creating and starting a stream")
     var lastPartStart = System.nanoTime()
-    val future = Source(Stream.from(0, 1))
```
Comment (drewhk) [inline benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaProducerBenchmarks.scala]: We really would need some good support for static code analysis. Stream should be not used ever and there should be ways to enforce this automatically.

**Comment by drewhk** [comment]
LGTM, great find!

File: benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaProducerBenchmarks.scala (hunk 1 above)
Comment (kciesielski) [inline benchmarks/src/main/scala/akka.kafka.benchmarks/ReactiveKafkaProducerBenchmarks.scala]: Learning a lot with you guys :) Thanks for spotting this.

**Comment by patriknw** [comment]
it's a great exchange in both directions :-)

**Pull request merged**